packages/terra-clinical-item-display/src/ItemDisplay.jsx

```diff
@@ -35,14 +37,21 @@ const propTypes = {
    * The icon react element to be displayed next to the display text.
    */
   icon: PropTypes.element,
+  /**
+   * The position of the icon to be displayed next to the text,
+   * noticeable when the text display wraps multiple line.
+   * One of `'center'`, `'top'`, `'inline'`.
+   */
+  iconAlignment: PropTypes.oneOf(['center', 'top', 'inline']),
 };
 
 const defaultProps = {
   text: '',
-  textStyle: undefined,
+  textStyle: 'primary',
```

Comment:
By changing this default value this will be non-passive. Is that intended?

Reply:
It actually should still passive since the color remains the same and primary was already assumed as the default, but you could never explicitly set it or test for it, or re-theme it.

Reply:
Fixed: https://github.com/cerner/terra-clinical/pull/490/commits/c86aab122395aef6137dbf691f69476d0949ab57#diff-7a7f3d6691e06f96790e846013f347ff